### ISSUE TITLE
Quest tagging

### ISSUE THREAD

**Issue by Rumik** [dcaslin/d2-checklist#161]
Would it be possible to create a similar tagging system for quests, and then combine them? If I have a quest step involving the crucibal, for example, I’d want to see it grouped with crucibal bounties on the home screen. Or if that step of the quest involves a scout rifle, I’d want it to appear with scout rifle bounties.

Ideally, whether I’m looking at a bounty or a quest step, I’d see the associated tags and an indication of the number of other pursuits that have the same tag, so I could easily find other pursuits that synergise with the one I’m looking at.

That’s the dream! :D

**Comment by Rumik** [comment]
Did anything ever happen with this idea?

**Comment by dcaslin** [comment]
I couldn't get it working well

**Comment by dcaslin** [comment]
I couldn't get it working... but now that I started typing you an answer... I don't think it would work well for "Sets for Sale" but I guess it might work for "Sets Currently Held", I'd need to figure out how to run through Quests in addition to bounties. When I browsed through Quests my chars held, I found they were all too specific to tag well and/or they had high enough numbers that I felt like they might get irritating (1000 bow kills for hush would just be sitting out there forever). Thoughts?
